chapters/data_processing/transform_images.ipynb

```diff
@@ -0,0 +1,4003 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import tensorflow as tf\n",
+    "import IPython.display as display\n",
+    " \n",
+    "raw_image_dataset = tf.data.TFRecordDataset('./data/tfrecords_small_tfrecord-00000-of-00001')\n",
```

Comment:
Do you mind renaming the file to `tfrecords_small.tfrecord-00000-of-00001` to follow the naming convention from Google tutorials (https://www.tensorflow.org/tutorials/load_data/tfrecord#tfrecords_format_details)?

Reply:
Ok